apps/browser/src/autofill/overlay/inline-menu/pages/list/autofill-inline-menu-list.ts

```diff
@@ -1174,13 +1174,24 @@ export class AutofillInlineMenuList extends AutofillInlineMenuPageElement {
     return cipherDetailsElement;
   }
 
+  /**
+   * Checks if there is more than one TOTP element being displayed.
+   *
+   * @returns {boolean} - Returns true if more than one TOTP element is displayed, otherwise false.
+   */
+  private multipleTotpElements(): boolean {
+    return (
+      this.ciphers.filter((cipher) => cipher.login?.totpField && cipher.login?.totp).length > 1
+    );
+  }
+
   /**
    * Builds a TOTP element for a given TOTP code.
    *
    * @param totp - The TOTP code to display.
    */
 
-  private buildTotpElement(totpCode: string): HTMLDivElement | null {
+  private buildTotpElement(totpCode: string, username: string): HTMLDivElement | null {
```

Comment:
nit; since username is not a cipher requirement:

```suggestion
  private buildTotpElement(totpCode: string, username?: string): HTMLDivElement | null {
```

Reply:
good call!